### ISSUE TITLE
Enabling a discussion for the design direction of this repo

### ISSUE THREAD

**Issue by bjl25** [IDAES/pse-home#1]
For context, the Ahuora team needs a compiled list of the tools PSE+ has developed and where to find them as we are trying to create a development roadmap for our platform. This has been a difficult challenge as tools and functionality are spread across unlinked repos and documentation (or the repo is so large the functionality is lost).

My personal view is that this repo can hopefully be a stepping stone to a central place where people can find the functional tools they need. I am not sure how we should tackle this problem but I might try a layout today.

PS: This kind of collaboration might work better in a discussion but I am not sure how your team wants to operate.

**Comment by andrewlee94** [comment]
@bjl25 To fill you in on some details of what is happening internally right now, we are trying to organize a cross-project team to discuss unifying all the documentation into a single, central location (which may well end up here). This is still in the trying-to-organise a first meeting stage though.

There has also been some talk, both historically and recently, of trying to maintain a register of some sort for documenting existing capabilities and those under development. We have tried in the past, but the issue we ran into was that no-one bothered to check the register before starting development, and thus never added their new capabilities to the list either. Unfortunately, this really is one of those things where we have to rely almost exclusively on the developers to engage, as there is no way that we could assign a central custodian who would mysteriously know when a new capability began development. I am pushing to require that all new internal developments go throguh a formally planning stage before coding begins that would help with this, but again we would need to see if people actually engage with it. I did just do a quick check of our internal Box folder and found our last attempt at this - whilst it will only be viewable to team members, I am posting the link here for reference: https://app.box.com/file/878116106557?s=vdzhr1j1g0ec6bia5sh9tdxb315sd4ux

Any thoughts or suggestions you might have would be appreciated.

**Comment by ksbeattie** [comment]
This repo is superseded by the https://github.com/idaesplus org.
